addon/components/s-modal.js

```diff
@@ -1,8 +1,18 @@
 import Component from '@ember/component';
 import layout from '../templates/components/s-modal';
+import {
+  computed
+} from '@ember/object';
 
 export default Component.extend({
-    layout,
-    classNames: ['s-modal']
+  layout,
+  classNames: ['s-modal'],
+  actionPassed: computed('onCancel', 'onOK', function(){
```

Comment:
Can you rename this property to `showFooter` please because that's what this property does.

Reply:
Done @lalitindoria